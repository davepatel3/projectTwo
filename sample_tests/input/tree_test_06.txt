string
peek_nodes
allocate
vB(9Jf
0
allocate
X`\e.O2xqe
1
ref_node
2
set_parent
2
0
allocate
vLaj>2J9[2_Y71YC
0
remove
3
allocate
os?*;s
1
remove
1
remove
2
allocate
{*,0.n"gaF#>O]
0
allocate
s;saZs7w4sR:./"b
0
allocate
1sox=D
0
remove
2
remove
1
ref_node
3
remove
3
allocate
wdW{ZJt
0
ref_node
2
allocate
`BSNC
2
remove
2
allocate
zL;ZS+~bL2]uX
0
allocate
OpvMe-as8"&}V++
1
peek_nodes
remove
3
allocate
/Fz}N,\AP>]6-1hH
0
ref_node
3
allocate
)wR*
0
remove
2
remove
1
allocate
;G'?8wy_82(~l0f
0
allocate
xJbW@yOf
3
set_parent
2
0
set_parent
2
3
ref_node
1
ref_node
3
allocate
$Z}qzdMki\Uoge
3
allocate
qU1cG1
0
set_parent
1
2
allocate
o(tKMkqvUW
1
set_parent
4
3
remove
3
allocate
vsUOy6D~{AAQ
0
allocate
If/zk8mFuGPkg!a
5
peek_nodes
ref_node
1
allocate
l$d7..e@;E.x\2Z
5
allocate
CRi(I5
1
remove
5
allocate
^D;h.
0
allocate
v&,$y94xA4}Np~y
3
ref_node
4
set_parent
4
0
allocate
w-jipSG
4
ref_node
3
remove
3
allocate
};l<
1
allocate
V`{+C9lO
4
allocate
:s#z&
2
allocate
9Mx0(Wj)rzLi6L;*
3
peek_nodes
set_parent
2
4
remove
3
allocate
d4.j"Eu
2
allocate
Wzq&ec]*V
4
allocate
_#5d
6
remove
2
allocate
V-ZNCH5X#!)TF
0
set_parent
5
0
remove
7
remove
1
allocate
2G!pNV{G/D"
4
peek_nodes
allocate
U\_7=eVVS
0
ref_node
2
ref_node
5
allocate
>*ax"T^U}'N
2
ref_node
4
ref_node
5
allocate
g4p_Y
6
allocate
v]>Ej+*b+P7vCZ6
7
remove
3
allocate
/%UzK
2
allocate
crrZs`fK2(
3
ref_node
2
allocate
.4U#L@#im{?
7
remove
5
allocate
k0K[u5JXk)=i@VD
10
ref_node
7
remove
10
allocate
(8~yI~r<DAFWIe
0
remove
5
remove
7
set_parent
3
0
set_parent
1
2
allocate
y~jYzktSLrU
2
allocate
D#VT
1
allocate
}vI%4w1
5
remove
9
allocate
}0WjOyIrp`&0H
5
peek_nodes
allocate
`d=G1d(P7Q(
5
set_parent
4
7
set_parent
3
4
set_parent
4
5
remove
10
ref_node
6
peek_nodes
remove
8
peek_nodes
allocate
xuYOHzc(u
6
remove
6
remove
3
allocate
1s]i3W6Y
2
allocate
x\t"J~:<
2
allocate
x6@nq#
5
set_parent
7
10
remove
8
allocate
hQj9O;Q$f};
6
allocate
@1-W6(^x{Q
1
allocate
)da_aYG]b
5
ref_node
6
set_parent
7
3
allocate
"g%wdrPr
5
set_parent
8
17
allocate
`T0aI$*
11
peek_nodes
remove
6
allocate
}(x*-L'\_)7Lw
4
allocate
Ujd8
9
ref_node
10
peek_nodes
remove
12
allocate
(;Ui0.6%O}XJw\
5
remove
2
allocate
#w*T@_{'_Kx
0
allocate
WbI*68[1gaZ?6rm5
12
remove
3
remove
12
quit
