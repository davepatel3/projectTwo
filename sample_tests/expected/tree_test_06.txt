peek_nodes
0: 
allocate
1
allocate
2
ref_node
X`\e.O2xqe
parent: 1
children: 
set_parent
allocate
3
remove
allocate
3
remove
remove
allocate
3
allocate
1
allocate
2
remove
remove
ref_node
{*,0.n"gaF#>O]
parent: 0
children: 
remove
allocate
2
ref_node
wdW{ZJt
parent: 0
children: 
allocate
1
remove
allocate
3
allocate
Node is recycled!
peek_nodes
0: 
1: [recycled]
2: [recycled]
3: zL;ZS+~bL2]uX
remove
allocate
1
ref_node
Node is recycled!
allocate
2
remove
remove
allocate
3
allocate
2
set_parent
set_parent
ref_node
Node is recycled!
ref_node
;G'?8wy_82(~l0f
parent: 0
children: 2 
allocate
1
allocate
4
set_parent
allocate
5
set_parent
remove
allocate
5
allocate
1
peek_nodes
0: 
1: If/zk8mFuGPkg!a
2: [recycled]
3: [recycled]
4: [recycled]
5: vsUOy6D~{AAQ
ref_node
If/zk8mFuGPkg!a
parent: 5
children: 
allocate
2
allocate
4
remove
allocate
3
allocate
4
ref_node
v&,$y94xA4}Np~y
parent: 3
children: 
set_parent
allocate
1
ref_node
^D;h.
parent: 0
children: 
remove
allocate
2
allocate
5
allocate
3
allocate
6
peek_nodes
0: 
1: w-jipSG
2: };l<
3: :s#z&
4: v&,$y94xA4}Np~y
5: V`{+C9lO
6: 9Mx0(Wj)rzLi6L;*
set_parent
remove
allocate
6
allocate
3
allocate
7
remove
allocate
7
set_parent
remove
remove
allocate
6
peek_nodes
0: 
1: [recycled]
2: [recycled]
3: Wzq&ec]*V
4: v&,$y94xA4}Np~y
5: V`{+C9lO
6: 2G!pNV{G/D"
7: [recycled]
allocate
2
ref_node
U\_7=eVVS
parent: 0
children: 
ref_node
V`{+C9lO
parent: 0
children: 
allocate
7
ref_node
v&,$y94xA4}Np~y
parent: 0
children: 3 6 
ref_node
V`{+C9lO
parent: 0
children: 
allocate
1
allocate
8
remove
allocate
3
allocate
9
ref_node
U\_7=eVVS
parent: 0
children: 7 3 
allocate
10
remove
allocate
5
ref_node
>*ax"T^U}'N
parent: 2
children: 8 10 
remove
allocate
5
remove
remove
set_parent
set_parent
allocate
10
allocate
5
allocate
8
remove
allocate
7
peek_nodes
0: 
1: g4p_Y
2: U\_7=eVVS
3: /%UzK
4: v&,$y94xA4}Np~y
5: D#VT
6: 2G!pNV{G/D"
7: }0WjOyIrp`&0H
8: }vI%4w1
9: [recycled]
10: y~jYzktSLrU
allocate
9
set_parent
set_parent
set_parent
remove
ref_node
2G!pNV{G/D"
parent: 4
children: 
peek_nodes
0: 
1: g4p_Y
2: U\_7=eVVS
3: /%UzK
4: v&,$y94xA4}Np~y
5: D#VT
6: 2G!pNV{G/D"
7: }0WjOyIrp`&0H
8: }vI%4w1
9: `d=G1d(P7Q(
10: [recycled]
remove
peek_nodes
0: 
1: g4p_Y
2: U\_7=eVVS
3: /%UzK
4: v&,$y94xA4}Np~y
5: D#VT
6: 2G!pNV{G/D"
7: }0WjOyIrp`&0H
8: [recycled]
9: `d=G1d(P7Q(
10: [recycled]
allocate
10
remove
remove
allocate
8
allocate
10
allocate
6
set_parent
remove
allocate
3
allocate
8
allocate
11
ref_node
x6@nq#
parent: 5
children: 3 
set_parent
allocate
12
set_parent
Invalid handle!
allocate
13
peek_nodes
0: 
1: g4p_Y
2: U\_7=eVVS
3: hQj9O;Q$f};
4: v&,$y94xA4}Np~y
5: D#VT
6: x6@nq#
7: }0WjOyIrp`&0H
8: @1-W6(^x{Q
9: `d=G1d(P7Q(
10: x\t"J~:<
11: )da_aYG]b
12: "g%wdrPr
13: `T0aI$*
remove
allocate
7
allocate
3
ref_node
x\t"J~:<
parent: 2
children: 
peek_nodes
0: 
1: g4p_Y
2: U\_7=eVVS
3: Ujd8
4: v&,$y94xA4}Np~y
5: D#VT
6: [recycled]
7: }(x*-L'\_)7Lw
8: @1-W6(^x{Q
9: `d=G1d(P7Q(
10: x\t"J~:<
11: )da_aYG]b
12: "g%wdrPr
13: `T0aI$*
remove
allocate
6
remove
allocate
12
allocate
3
remove
remove
quit
